infrastructure/sandbox.tfvars

```diff
@@ -0,0 +1,2 @@
+capacity = "2"
+elasticsearch_hosts = "http://10.112.0.4:9200"
```

Comment:
this is hardcoded for now until we finalise deployment of Elastic to azure which at the moment is still a spike